jobs/rocoto/aeroanlfinal.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -x
```

Comment:
These should all use `#! /usr/bin/env bash` to be consistent with existing scripts. Also, the first line afterwards should be `source "$HOMEgfs/ush/preamble.sh"`.

Reply:
ok, will update tomorrow. FYI there are several .sh files in this directory in `develop` that do not meet this convention.

Reply:
Yeah, those came in in a recent PR that was merged before I got a chance to say anything. I need to fix them.

Reply:
They are the JEDI-atm ones I believe, I can include those in this PR if you would like, otherwise we can do them separately.

Reply:
It's a minor change and you'll be doing the others anyway, might as well include them here.

Reply:
@WalterKolczynski-NOAA see if the latest commit addresses this request properly